sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -8,6 +8,7 @@
 #include "azure/core/http/pipeline.hpp"
 #include "azure/core/nullable.hpp"
 #include "azure/core/response.hpp"
+#include "azure/storage/common/crypt.hpp"
```

Comment:
Why did we add this header? Is it needed?

Reply:
Since Core does not encode the query parameter value any more Storage SDK has to add this. However, we can discuss more on where the best place is for this helper function.
Let's wait until Jinming is back to discuss more in issue: https://github.com/Azure/azure-sdk-for-cpp/issues/736